moped-editor/src/constants/projects.js

```diff
@@ -0,0 +1,2 @@
+export const substantialCompletionDateTooltipText =
+  "The earliest confirmed start date of a project phase that has a simple phase name of Complete (Complete or Post-construction)";
```

Comment:
i love the tooltip, but i wonder if users know what "simple phase" is, since it's not really mentioned in the app anywhere. maybe better to not mention it at all, e.g

>  The earliest confirmed start date of any Complete or Post-construction phase

Reply:
oh, interesting—when testing this just now it appears that the substantial completion date looks at both the start and end dates of a phase. so the truth is:

> The earliest confirmed ~start~ date of any Complete or Post-construction phase

Reply:
@chiaberry thank you for creating this constants file!